### PR TITLE
Project improvements (working in progress)

### PR DESCRIPTION
**Pull request by thiamsantos**
As discussed on #9, I am adding some things on the project. I will appreciate any comment and review on this pull request.

#### TODO

- [x] Improve README
- [x] Improve contributor guidelines
- [ ] Implement tests
- [ ] Achieve 100% code coverage with tests
- [x] Add a code style
- [ ] Integrate with [Travis CI](https://travis-ci.org/)
- [ ] Integrate with [Coveralls](https://coveralls.io/)
- [ ] Add [cross-browser support test](https://wiki.saucelabs.com/display/DOCS/Using+Status+Badges+and+the+Browser+Matrix+Widget+to+Monitor+Test+Results) using [Sauce Labs](https://saucelabs.com/)
- [ ] Add [Git Hooks](https://git-scm.com/docs/githooks) to prevent bad commits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,83 @@
...
+$ bower install progressively
 ```
-bower install progressively
-npm install progressively
+
+Then with a module bundler like [rollup](http://rollupjs.org/) or [webpack](https://webpack.js.org/), use as you would anything else:
+
+```javascript
+// using ES6 modules
+import mitt from 'mitt'
```
Comment (thinker3197) [inline README.md]: Use progressively name as example.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,83 @@
...
+
+// using CommonJS modules
+var mitt = require('mitt')
```
Comment (thinker3197) [inline README.md]: Use progressively name as example

File: README.md (hunk 1 above)
Comment (thiamsantos) [reply README.md]: Sorry, my bad

**Review by thinker3197 — APPROVED** [review]
This looks good, but please make the above changes @thiamsantos!

**Comment by thiamsantos** [comment]
Thanks for merging :tada: :tada:

I will continue sending changes in another pull request.

**Pull request merged**